### PR TITLE
Use latest o-forms (1.0.0-beta.8)

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Fixes https://jira.ft.com/browse/NFT-26

Would be great if the OBT demo task understood handlebars, then we could reuse the app's templates

### CODE REVIEW

File: demos/src/default-panel.mustache [hunk 1]
```diff
@@ -21,9 +21,9 @@
...
 			<div class="o-header__secondary o-header__secondary--search">
-				<form action="/search" class="o-ft-forms__affix-wrapper">
-					<input type="search" name="q" class="o-ft-forms__field" placeholder="Search here" value="{{searchQuery|default('')}}"/>
-					<div class="o-ft-forms__suffix">
+				<form action="/search" class="o-forms-affix-wrapper">
+					<input type="search" name="q" class="o-forms-text" placeholder="Search here" value="{{searchQuery|default('')}}"/>
+					<div class="o-forms-suffix">
 						<button class="o-ft-buttons">Go</button>
 					</div>
 				</form>
```
Comment (ironsidevsquincy) [inline demos/src/default-panel.mustache]: wonder if these two are still needed, demo pages didn't really display anything

**Comment by ironsidevsquincy** [comment]
idiot! https://github.com/Financial-Times/next-header/pull/69

**Comment by matthew-andrews** [comment]
@ironsidevsquincy the handlebars debate is a long running one without a proper conclusion yet.

tl;dr we need a solution that transcends javascript (so mustache has been chosen as the least worst option for now)

https://github.com/Financial-Times/ft-origami/issues/310

**Comment by wheresrhys** [comment]
@ironsidevsquincy I'll merge this in. A vague idea I had re handlebars vs mustache is a task (in next-build-tools?) that renders the template using ft-next-express's extended version of handlebars and saves it to demos/src/main.mustache. The output would be raw html, which is valid mustache, so then `next-build-tools demo && origami-build-tools demo` should work. 

Had no time to do anything about it though

**Comment by wheresrhys** [comment]
in fact, no, gonna merge into my branch firs then in to master in one go
